executor/api/kafka/proxy.go

```diff
@@ -0,0 +1,191 @@
+package kafka
+
+import (
+	"context"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/go-logr/logr"
+	"github.com/seldonio/seldon-core/executor/api/client"
+	"github.com/seldonio/seldon-core/executor/api/payload"
+	"github.com/seldonio/seldon-core/executor/api/rest"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+type KafkaProxy struct {
```

Comment:
In what context would the proxy be used instead of the server? It seems it's specific to models instead of graph level, but there's already the RPC topics.go in place which is called in the kafkaclient, but this doesn't seem to be called from other places other than the makefile

Reply:
proxy is only used for kafka_full_graph option so is not needed when the graph is calling using http/grpc